app/packages/forms/controllers/form_results.coffee

```diff
@@ -0,0 +1,28 @@
+Template.form_results.onCreated ->
+  @selectedFormIdCollection = new Meteor.Collection null
+  @selectedFormIds = new ReactiveVar []
+
+Template.form_results.onRendered ->
+  instance = @
+  @autorun =>
+    _selectedFormIds =
+      _.map @selectedFormIdCollection.find().fetch(), (form) -> form.id
```

Comment:
I think we'll still need to use pluck because we need an array of just the ids. But I suppose its good practice to limit the properties returned from the query.